src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKAnnotationPluginDescriptor.java

```diff
@@ -497,4 +496,12 @@ public Class<?> getClassForPluginHelp(final String pluginName) {
         return allDiscoveredAnnotations.containsKey(pluginName) ? allDiscoveredAnnotations.get(pluginName).getClass() : null;
     }
 
+    public Map<String,Annotation> getAllDiscoveredAnnotations() {
```

Comment:
If we keep this method, I'd suggest:

- moving it so its adjacent to `getResolvedInstances`
- renaming it to `getDiscoveredInstances`
- adding a javadoc comment saying that it returns *every* annotation that is discovered anywhere on the classpath, and that it should only be used when `getResolvedInstances` isn't sufficient